flake.nix

```diff
@@ -218,32 +229,45 @@
 
           hc-scaffold =
             craneLib.buildPackage {
-                pname = "lair-keystore";
-                src = craneLib.cleanCargoSource inputs.hc-scaffold;
+              pname = "hc-scaffold";
+              src = craneLib.cleanCargoSource inputs.hc-scaffold;
 
-                doCheck = false;
+              doCheck = false;
 
-                buildInputs = [
-                    pkgs.go
-                    pkgs.perl
-                ];
+              buildInputs = [
+                pkgs.go
+                pkgs.perl
+              ];
             };
         in
         {
+          # Configure a formatter so that `nix fmt` can be used for format this file.
+          formatter = pkgs.nixpkgs-fmt;
 
           packages = {
             inherit holochain;
             inherit lair-keystore;
-            inherit rust;
             inherit hc-launch;
             inherit hc-scaffold;
+            inherit rust;
+          };
+
+          # Define runnable applications for use with `nix run`.
+          # These can be used like `nix run "github:holochain/holonix#hc-scaffold" -- --version`.
+          # https://flake.parts/options/flake-parts.html?highlight=perSystem.apps#opt-perSystem.apps
+          apps = {
+            holochain.program = "${holochain}/bin/holochain";
+            lair-keystore.program = "${lair-keystore}/bin/lair-keystore";
+            hc-launch.program = "${hc-launch}/bin/hc-launch";
+            hc-scaffold.program = "${hc-scaffold}/bin/hc-scaffold";
```

Comment:
This means we can scaffold without needing to download binaries for everything that's in the Holonix shell. As long as it's cached of course! This will reduce the download size to get started.